registration/models.py

```diff
@@ -422,7 +422,7 @@ def _activate(self, profile, get_profile):
         """
 
         if not profile.user.is_active and not profile.activated:
-            site = Site.objects.get_current()
+            site = get_current_site()
```

Comment:
I think that https://github.com/macropin/django-registration/blob/master/registration/admin.py#L37 can also be changed to use the `get_current_site` shortcut